src/parse_constants.h

```diff
@@ -246,8 +246,8 @@ void parse_error_offset_source_start(parse_error_list_t *errors, size_t amt);
 
 /// Error message for wildcards with no matches.
 #define WILDCARD_ERR_MSG                                                                     \
-    _(L"No matches for wildcard '%ls'.  (Tip: empty matches are allowed in 'set', 'count', " \
-      L"'for'.)")
+    _(L"No matches for wildcard pattern '%ls'.\n"                                            \
```

Comment:
Frankly the existing Tip drives me nuts. A tip seems appropriate if it either only shows a few times, or if it always indicates a user error. But failed wildcard expansions are very frequent, so I don't think any tip is appropriate here.

I would like to drop the Tip and simply say `"No matches for wildcard pattern '%ls'.`